content/en/getting-started/api-key.md

```diff
@@ -30,13 +30,15 @@ LocalStack expects your API key to be present in the environment variable `LOCAL
 {{< tabpane >}}
 {{< tab header="macOS/Linux" lang="shell" >}}
 export LOCALSTACK_API_KEY=<YOUR_API_KEY>
-localstack start -d
+localstack start
```

Comment:
I guess it makes sense to not start in detached mode here, but then we should also change it in line 36 (instructions for windows).

Reply:
This was made to be consistent with the pro instructions. I'll update the windows docs as well.